coding_systems/bnf/README

```diff
@@ -8,10 +8,18 @@ For more details, see our most popular blog post [0].
 We obtain the raw data from the BSA [1].  New releases are published annually, sometimes
 with unannounced mid-year updates.
 
+Download the latest release, which will be a zipfile named something like `20220701_1656687539491_BNF_Code_Information.zip`
+
 To import the data, run:
 
-        ./manage.py import_data coding_systems.bnf path/to/dir/
+    ./manage.py import_data coding_systems.bnf path/to/zip/file
+    --release <release_name> --valid-from YYYYMMDD --import-ref <ref>
 
+- `release` is a short label for this coding system release - the official version number/reference if there is one.
+- `valid-from` is the date that this release is valid from (NOT the date it is being
+  imported into OpenCodelists), in YYYYMMDD format
+- `import-ref` is an optional long-form reference field for any other potential
+  references or comments on this import if desired, e.g. full filenames
```

Comment:
Can you be more explicit here?  Given a filename like `20220701_1656687539491_BNF_Code_Information.zip`, how would I know what to use for `release` and `valid-from`?  And can you offer some more prescriptive advice for `import-ref` too?